converter.go

```diff
@@ -333,7 +335,7 @@ func arrowToValue(
 						if higherPrecision {
 							(*destcol)[i] = f
 						} else {
-							(*destcol)[i] = fmt.Sprintf("%f", f)
+							(*destcol)[i] = fmt.Sprintf("%g", f)
```

Comment:
%g will change format of the output in E notation. Better fix will be like "%.*f", scale of column,f. Please look in that way. Still %g won't guarantee prec,scale of a column output.